docs/docs/concepts/jobs.md

```diff
@@ -12,38 +12,18 @@ This is an API where you can:
 - Get result from a job
 
 
-If your custom application want to use dm-job, you have to create a JobHandler file for your job.
+If your custom application want to run a job using dm-job, you have to 
+1) Create a JobHandler python file for your job.
+2) Create a job entity for the job of type `dmss://WorkflowDS/Blueprints/Job`. The job entity you create must be available in dmss, for example by uploading your job entity with the document add DMSS endpoint.
```

Comment:
This sounds more coplicated than it is. If the entity is placed in the app folder, it will be done automatically, and you don't have to worry about DMSS